assets/sass/components/sign-in-with-google/_googlesitekit-sign-in-with-google-setup-module.scss

```diff
@@ -18,6 +18,37 @@
 
 .googlesitekit-plugin {
 
+	.googlesitekit-setup-cta-banner__svg-wrapper--sign-in-with-google-setup-cta {
+
+		@media (min-width: $bp-wpdb3) {
+			height: 100%;
+			position: relative;
+		}
+
+		svg {
+			display: block;
+			justify-self: center;
+			max-height: 180px;
+
+			@media (min-width: $bp-nonMobile) {
+				max-height: 210px;
+			}
+
+			@media (min-width: $bp-xlarge) {
+				justify-self: flex-end;
+				max-height: 274px;
+			}
+
+			@media (min-width: $bp-wpdb3) {
+				max-height: 300px;
+				position: absolute;
+				right: 0;
+				top: 50%;
+				transform: translateY(-50%);
+			}
```

Comment:
For bigger desktop screens it is better fitting when it is a bit more expanded, to fill empty space, especially 1920+ width when banners get stretched